### ISSUE TITLE
Performance of this decreased model

### ISSUE THREAD

**Issue by 00001101-xt** [crystal-method/Looking-to-Listen#2]
As one can tell that the model you used is much smaller than the original Look-to-Listen, including:

1. decreased 2 dilated convolutional layers of the audio NN
2. decreased 1 dilated convolutional layer of the visual NN
3. LSTM is smaller
4. FCs is smaller 

It makes sense because one may be lack of training data and computation power. But I was wondering is there any reported performance of this model? If so, more specifically, it would be nice for you to share some information like:

- the size of the training data you used
- training parameters like (batch size, learning rate, etc. In the original paper, 6 and 0.00003 are used respectively)
- ...

Thank you.
